### DISCUSSION TITLE
Typesense DocSearch

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6579] [Q&A] [answered]
Is there an open source docusaurus website which has Typesense implemented instead of Algolia? I would be curious to look how it was implemented.

**Comment by slorber** [comment, marked as answer]
The best is to ask author @jasonbosco
https://github.com/typesense/docusaurus-theme-search-typesense

**Reply by jasonbosco** [reply]
Thank you for the ping @slorber. 

Hi @GambitFPS, here's one example of an open source Docusaurus site that implements Typesense: https://sdk-docs.polygon.technology/docs/overview/

Here's the source code for that site: https://github.com/0xPolygon/polygon-edge-docs

It's fairly straight-forward to set this up following these instructions: https://typesense.org/docs/guide/docsearch.html

Are you running into any issues that I can help with?
